### DISCUSSION TITLE
Any difference between this and solid-ui

### DISCUSSION THREAD

**Discussion by SillyCoon** [hngngn/shadcn-solid#121] [General]
Hello! Does this project has any relation to https://www.solid-ui.com/ ?

**Comment by hngngn** [comment]
Of course not

**Reply by re1sub** [reply]
Hi, could you elaborate? Since I'm struggling to see the difference between this project and solid-ui. Thank you.

**Reply by hngngn** [reply]
Shadcn-solid use `new-york` style, support `UnoCSS`, use `Unovis` for chart while solid-ui use `default` style, the rest is still in wip. Choose what work best for you.

**Comment by lalatina11** [comment]
> Hello! Does this project has any relation to https://www.solid-ui.com/ ?

Yes. solid ui is not that good maintenance. solid ui still using tailwind v3, this shadcn-solid from @hngngn is using tailwind 4. thanks for create this masterpiece, this is so helpful @hngngn
